### PR TITLE
Confirmation dialog for removing stake

### PR DESCRIPTION
**Pull request by kimpers**
**Desktop**
image

**Mobile**
image

### CODE REVIEW

File: ts/pages/account/dashboard.tsx [hunk 1]
```diff
@@ -83,9 +84,11 @@ interface PendingAction {
     amount: number;
 }
 
-// TODO(johnrjj) - After going to a pool that has been staked in, hitting the 'Remove' button,
-// and running the transaction, the balances do not update, even after a good amount of time (30min+ etc)
-// Is there a way we can query web3 directly?? (tl;dr 'Pending' removes are flaky/not sure if they are working)
```
Comment (kimpers) [inline ts/pages/account/dashboard.tsx:88]: @johnrjj can you confirm that this is not an issue anymore? It works fine for me but I'm not sure I fully understand what the problem was.

File: ts/components/staking/remove_stake_dialog.tsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+        @media (max-width: 768px) {
+            min-height: 100vh;
+            width: 100vw;
+            margin: 0;
+            padding: 30px;
+
+            border: none;
+        }
+    }
+`;
```
Comment (kimpers) [inline ts/components/staking/remove_stake_dialog.tsx:163]: Mostly copy-pasted from `change_pool_dialog`, it is small enough that it didn't feel worth it creating a generic dialog component but if we start adding more of these we should.

File: ts/pages/account/dashboard.tsx (hunk 1 above)
Comment (johnrjj) [reply ts/pages/account/dashboard.tsx:88]: This must've been outdated -- everything looks good now! :)

File: ts/components/staking/remove_stake_dialog.tsx (hunk 2 above)
Comment (johnrjj) [reply ts/components/staking/remove_stake_dialog.tsx:163]: that's my preferred level of abstraction too 😄

**Review by johnrjj — APPROVED** [review]
LGTM -- 0x 4 lyfe!

**Pull request merged**